api/api.go

```diff
@@ -70,5 +72,21 @@ func checkEnvVarShareResponse(resp *http.Response) error {
 	if resp.StatusCode >= 200 && resp.StatusCode < 300 {
 		return nil
 	}
-	return fmt.Errorf("request to %s failed: status code should be 2xx (%d)", resp.Request.URL, resp.StatusCode)
+	respErr := fmt.Errorf("request to %s failed: status code should be 2xx (%d)", resp.Request.URL, resp.StatusCode)
+	respBody, err := io.ReadAll(resp.Body)
+	if err != nil {
+		return respErr
+	}
+	var respBodyJSON map[string]string
+	err = json.Unmarshal(respBody, &respBodyJSON)
+	if err != nil {
+		return respErr
```

Comment:
Even if we can't parse the response into JSON, we could still print it as plain text. It's still more useful than just the error code